packages/language-support/src/autocompletion/completionCoreCompletions.ts

```diff
@@ -767,7 +779,10 @@ function completeSymbolicName({
     // so target should be non-existent
     (ruleList.includes(CypherParser.RULE_renameUser) && afterToToken)
   ) {
-    return parameterSuggestions;
+    return parameterSuggestions.map((completionItem) => ({
+      label: completionItem.label,
+      kind: completionItem.kind,
+    }));
```

Comment:
Why do we need these mappings to label and kind? This is not doing anything I think?

Reply:
Used to backtick if needed parameters too (so had insertText: backtickIfNeeded(...label) too), but they could not be added with backticks yet. Looks like I forgot to remove the whole thing. 

Either way, since we're changing the parser-file for creating params, maybe we should just add backticks if needed here already, so once it's valid, we're good to go?